performance/src/api_performance_tests.ts

```diff
@@ -173,6 +173,20 @@ const getloadTests = (
       ),
       !!sD?.workspace.projectId,
     ),
+    test(
+      "Get Project Columns for Runs Table",
+      getRequest(
+        `/api/v1/projects/${sD?.workspace.projectId}/columns?table_type=TABLE_TYPE_RUN`,
+      ),
+      !!sD?.workspace.projectId,
+    ),
+    test(
+      "Get Project Columns for Experiments Table",
```

Comment:
is this the same endpoint as https://github.com/determined-ai/determined/blob/805f2f4ea5af73b2f141f33a7771a3c5e255e8d9/performance/src/api_performance_tests.ts#L196C44-L196C53?

is it going to slow down all the other performance tests while we fix it or has it been improved?

Reply:
It is effectively the same call. I removed it as only the performance for the new columns have been improved.